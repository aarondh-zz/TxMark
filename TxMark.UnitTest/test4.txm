Branches
		(if: $IsFirstBranch)[`$`IsFirstBranch is true] 
		(else-if:$IsSecondBranch)[this other thing is true]
		(else-if:$IsThirdBranch)[this other other thing is true] 
		(else:)[this is false]

(set: $fubar to 3.14159 * 42 / 2)

The answer to the above equation is: $fubar.

(each: $array as $foo) [The value is $foo ]

