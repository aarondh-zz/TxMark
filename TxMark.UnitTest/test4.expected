<p>Branches
$IsFirstBranch is true
</p><p>

</p><p>
The answer to the above equation is: 65.97339.
</p><p>
The value is one The value is two The value is three The value is four The value is five The value is six 
</p>
